packages/flutter_test/lib/src/controller.dart

```diff
@@ -699,6 +699,47 @@ abstract class WidgetController {
   ///
   /// Shorthand for `Scrollable.ensureVisible(element(finder))`
   Future<void> ensureVisible(Finder finder) => Scrollable.ensureVisible(element(finder));
+
+  /// Repeatedly scrolls the `scrollable` by `dScroll` until `finder` and
+  /// throws if `finder` is not found for maximum `timeout` times.
+  ///
+  /// Between each scroll, wait for `duration` time for settling.
+  ///
+  /// This is different from [ensureVisible] in that this allows looking for
+  /// `finder` that is not built yet, but requires to specify the scrollable.
+  ///
+  /// If the `scrollable` is infinite and `dScroll`
+  Future<void> scrollUntilVisible(
+    Finder finder,
+    Finder scrollable,
+    double dScroll, {
```

Comment:
nit: avoid abbreviation - perhaps just call this `delta`?.

We should also document that delta is assumed to be in the `AxisDirection` of the scrollable.

Reply:
Done